lib/suggestion-list.js

```diff
@@ -245,31 +253,49 @@ export default class SuggestionList {
       }
     } else {
       this.destroyOverlay()
-      this.activeEditor = editor
-      this.displayBufferPosition = bufferPosition
-      const marker = this.suggestionMarker = editor.markBufferRange([bufferPosition, bufferPosition])
-      this.overlayDecoration = editor.decorateMarker(marker, {type: 'overlay', item: this, position: 'tail'})
-      this.addBindings(editor)
+      if (editor) {
+        this.activeEditor = editor
+        this.displayBufferPosition = bufferPosition
+        const marker = this.suggestionMarker = editor.markBufferRange([bufferPosition, bufferPosition])
+        this.overlayDecoration = editor.decorateMarker(marker, {type: 'overlay', item: this.suggestionListElement, position: 'tail', class: 'autocomplete-plus'})
+        const editorElement = atom.views.getView(this.activeEditor)
+        if (editorElement && editorElement.classList) {
+          editorElement.classList.add('autocomplete-active')
+        }
+
+        process.nextTick(() => { this.suggestionListElement.didAttach() })
```

Comment:
I don't think so, but ideally we would do this in a `atom.views.updateDocument` callback. I guess it should be fine, because we still schedule an animation frame as part of `render()` which is called in `didAttach()`.